internal/controllers/topology/cluster/patches/variables/variables.go

```diff
@@ -127,6 +149,13 @@ func Global(clusterTopology *clusterv1.Topology, cluster *clusterv1.Cluster) (Va
 				Version: cluster.Spec.Topology.Version,
 				Class:   cluster.Spec.Topology.Class,
 			},
+			Network: &ClusterNetworkBuiltins{
+				APIServerPort:      cluster.Spec.ClusterNetwork.APIServerPort,
+				ServiceDomain:      cluster.Spec.ClusterNetwork.ServiceDomain,
+				ServicesCIDRBlocks: cluster.Spec.ClusterNetwork.Services.CIDRBlocks,
+				PodsCIDRBlocks:     cluster.Spec.ClusterNetwork.Pods.CIDRBlocks,
```

Comment:
These two can panic.

EDIT: All others too as ClusterNetwork is a pointer

Reply:
Let's please also have unit tests to make sure we don't panic if ClusterNetwork or the service/pod CIDRs are nil